packages/common/src/core/types/IHubEditableContent.ts

```diff
@@ -19,6 +20,16 @@ export interface IHubEditableContent
    * capability enabled. This is a pre-requisite for Hosted Downloads to work.
    */
   serverExtractCapability?: boolean;
+  /**
+   * If the item represents a hosted feature service with "Extract enabled", shows the formats that
+   * can be extracted from the service via the "createReplica" operation.
+   */
+  serverExtractFormats?: string[];
```

Comment:
Does it make any sense to specify that the type is replica format array (eg `createReplicaFormat[]`)?

Reply:
@tannerjt It's a good question, and I'm torn. The unfortunate reality is that `CreateReplicaFormat` is an incomplete list, it's just the ones we know about. My thought was that we would leave it as `string[]` here to indicate our uncertainty of what will actually be returned back.

The next question may be "why have `CreateReplicaFormat` at all if we're not sure they are the right values?" Even though we are uncertain, I think it is worth having a list within the downloads sub-package that developers deep in the bowels of downloads can use as a sanity check. But what do you think?

Reply:
It seems perfectly okay to leave it as-is for now.